administrator/components/com_joomgallery/views/maintenance/view.html.php

```diff
@@ -215,6 +215,11 @@ function display($tpl = null)
       }
     }
 
+    // Additional Tabs added by plugins
+    $this->event  = new stdClass();
+    $addTabs  = $this->_mainframe->triggerEvent('onJoomAfterDisplayTabs', array(_JOOM_OPTION.'.maintanance',$tab));
```

Comment:
According to https://www.joomgalleryfriends.net/plugin-events.html#onJoomAfterDisplayTabs "$addTabs" has the type "string" but implode('', $addTabs) needs an array.